client/my-sites/stats/features/modules/stats-locations/use-option-labels.ts

```diff
@@ -0,0 +1,34 @@
+import { useTranslate } from 'i18n-calypso';
+import {
+	STATS_FEATURE_LOCATION_REGION_VIEWS,
+	STATS_FEATURE_LOCATION_COUNTRY_VIEWS,
+	STATS_FEATURE_LOCATION_CITY_VIEWS,
+} from '../../../constants';
+import { OPTION_KEYS } from './types';
+
+export default function useOptionLabels() {
```

Comment:
💯

Reply:
It could use a useMemo here. Will chunk in my next PR🙂